### ISSUE TITLE
Feature request: support completion for sub-commands like switch, delete, rename

### ISSUE THREAD

**Issue by nekomeowww** [sunny0826/kubecm#787]
## Is your feature request related to a problem? Please describe.

It is good to have such config management tool, how ever the current completion and cli design and implementation requires user to interactively navigate the options around when using sub-commands like`switch`, `delete`, `rename`. However, using both of interactive cli and completion is a bit of splitting my mind when native `kubectl config use-context` provides the cli interactive way with completion. Is it ok to provide both completion and interactive ways to select the target context? I'm happy to create and implement this feature.

## Describe the solution you'd like

Provide both completion and interactive ways to select the target context.

## Describe alternatives you've considered

None.

**Comment by nekomeowww** [comment]
Yes. Does it? I do know that I could use `kubecm switch minikube` and `kubecm switch dev` to explicitly pass the context name argument in. But I was talking about implement the shell completion for these sub-commands (Not sure if we are aligned about the topic). I failed to ask it to prompt the completion for context names eventually. Did I missed any configurations and documentations?

**Comment by sunny0826** [comment]
Oh, not really, I misunderstood you.
